rktl_control/launch/mean_odom_filter.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+  <node pkg="rktl_control" type="mean_odom_filter" name="filter" output="screen">
+    <rosparam command="load" file="$(find rktl_control)/config/mean_odom_filter.yaml" />
+  </node>
+</launch>
```

Comment:
Rather than having many nearly empty launch files, I think we should do something like what perception did and have a launch file that runs the whole stack:

- particle or odom filter
- velocity controller

I guess the ball uses just a filter. But rather than `mean_odom_filter.yaml`, we should have a ball and a car one.